bin/v-change-web-domain-proxy-tpl

```diff
@@ -71,6 +72,7 @@ fi
 # Add new vhost
 PROXY="$template"
 PROXY_EXT="$extentions"
+PROXY_URL="$proxy_url"
```

Comment:
Needs validation this is an security leak hole to be happening
Also we need a way to setup port number